dependent_type_theory.md

```diff
@@ -1,9 +1,9 @@
 Dependent Type Theory
 =====================
 
-Dependent type theory is a powerful and expressive language, allowing
+*Dependent type theory* is a powerful and expressive language, allowing
 you to express complex mathematical assertions, write complex hardware
-and software specifications, and reason about both of these in a
+and software specifications, and reason about all of these in a
```

Comment:
"both" seems correct to me here - it's referring back to two things (assertions and specifications).

Reply:
Just to spell out the rationale for the suggestion, since the preceding text is "(a and (b and c))", the "both" causes the reader to do a bit more mental work to check the correspondence.